Source/WebCore/Modules/webcodecs/WebCodecsAudioDecoder.cpp

```diff
@@ -124,7 +134,7 @@ ExceptionOr<void> WebCodecsAudioDecoder::configure(ScriptExecutionContext&, WebC
             return WebCodecsControlMessageOutcome::Processed;
         }
 
-        Ref createDecoderPromise = AudioDecoder::create(config.codec, createAudioDecoderConfig(config), [identifier, weakThis = ThreadSafeWeakPtr { *this }, decoderCount = ++m_decoderCount] (auto&& result) {
+        Ref createDecoderPromise = AudioDecoder::create(codec, config, [identifier, weakThis = ThreadSafeWeakPtr { *this }, decoderCount = ++m_decoderCount] (auto&& result) {
```

Comment:
It might be a bit more efficient if we would WTFMove(config).
That would remove potential copies of Config, hence the description vector as well.

Reply:
> It might be a bit more efficient if we would WTFMove(config). That would remove potential copies of Config, hence the description vector as well.

none of the Audio/VideoDecoder implementation use move semantics. they take a const reference and perform copies. I could probably amend that, but it will be a bigger change

Reply:
I'll take this in a follow up change